pkg/objstore/azure/azure.go

```diff
@@ -24,13 +27,63 @@ const (
 	azureDefaultEndpoint = "blob.core.windows.net"
 )
 
+// Set default retry values to default Azure values. 0 = use Default Azure
+var DefaultConfig = Config{
+	PipelineConfig: PipelineConfig{
+		MaxTries:      0,
+		TryTimeout:    0,
+		RetryDelay:    0,
+		MaxRetryDelay: 0,
+	},
+	ReaderConfig: ReaderConfig{
+		MaxRetryRequests: 0,
+	},
+	HTTPConfig: HTTPConfig{
```

Comment:
Can we reuse this struct?

Reply:
Yes, we should. See my other comments about not doing it in this PR, but create a new issue/PR for this

Reply:
We could, but why? (:

Reply:
Cool let's add in https://github.com/thanos-io/thanos/issues/4104 or so